lib/disability_compensation/providers/document_upload/lighthouse_supplemental_document_upload_provider.rb

```diff
@@ -61,7 +61,14 @@ def validate_upload_document(lighthouse_document)
   # @param file_body [String]
   def submit_upload_document(lighthouse_document, file_body)
     log_upload_attempt
-    api_response = BenefitsDocuments::Form526::UploadSupplementalDocumentService.call(file_body, lighthouse_document)
+
+    begin
+      api_response = BenefitsDocuments::Form526::UploadSupplementalDocumentService.call(file_body, lighthouse_document)
+    rescue => e
```

Comment:
This and the error handling that follows is the primary fix in this PR; note these changes only cover the sad path, the happy path that is currently working for most uploads in production should be unaffected and this PR should be low risk even though these changes aren't directly behind a flipper.